pkgs/build-support/docker/store-path-to-layer.sh

```diff
@@ -30,12 +30,13 @@ mkdir -p nix/store
 # store path to the absolute store path.
 tarhash=$(
   basename -a "$@" |
-    tar -cp nix \
+    tar -cpP nix \
```

Comment:
Can we use long flags here? I don't know what this means without looking it up in the man page.

Reply:
Right, good idea.